portfolio/src/test/java/com/google/sps/CommentsServletPostTest.java

```diff
@@ -0,0 +1,189 @@
+package com.google.sps;
```

Comment:
You can merge this file into _CommentsServletGetTest.java_ since they test the same class and call it _CommentsServletTest_

Reply:
Yes, I just thought that separating this test will provide a better readability, also they have a little bit different private fields because smth that is required in POST request isn't needed for GET and vice versa